src/modules/powerrename/PowerRenameUILib/MainWindow.xaml.cpp

```diff
@@ -150,13 +233,10 @@ namespace winrt::PowerRenameUI::implementation
                     if (SUCCEEDED(m_prManager->Advise(&m_managerEvents, &m_cookie)))
                     {
                         CComPtr<IShellItemArray> shellItemArray;
-                        // To test PowerRename uncomment this line and update the path to
-                        // your local (absolute or relative) path which you want to see in PowerRename
-                        // g_files.push_back(<path>);
-
+                        // To test PowerRename uncomment DEBUG_BENCHMARK_1M_ENTRIES define
```

Comment:
```suggestion
                        // To test PowerRename uncomment DEBUG_BENCHMARK_100K_ENTRIES define
```

Reply:
oh yeah, thanks found that 1M is too long to create